transit_odp/fares_validator/views/functions.py

```diff
@@ -400,86 +400,6 @@ def is_name_present_in_fare_frame(context, fare_zones, *args):
                     return response
 
 
-def is_members_scheduled_point_ref_present_in_fare_frame(context, fare_zones, *args):
-    """
-    Check if fareZones is present in FareFrame.
-    If true, then fareZones properties (members and ScheduledStopPointRef) should be present.
-    """
-    if fare_zones:
-        xpath = "../x:TypeOfFrameRef"
-        type_of_frame_ref = fare_zones[0].xpath(xpath, namespaces=NAMESPACE)
-        if type_of_frame_ref:
-            try:
-                type_of_frame_ref_ref = _extract_attribute(type_of_frame_ref, "ref")
-            except KeyError:
-                return ""
-            if not (
-                type_of_frame_ref_ref is not None
-                and (
-                    TYPE_OF_FRAME_REF_FARE_ZONES_SUBSTRING in type_of_frame_ref_ref
-                    or TYPE_OF_FRAME_REF_SERVICE_FRAME_SUBSTRING
-                    in type_of_frame_ref_ref
-                )
-            ):
-                sourceline_type_of_frame_ref = type_of_frame_ref[0].sourceline
-                response_details = XMLViolationDetail(
-                    "violation",
-                    sourceline_type_of_frame_ref,
-                    msg.MESSAGE_OBSERVATION_FARE_FRAME_TYPE_OF_FRAME_REF_REF_MISSING,
-                )
-                response = response_details.__list__()
-                return response
-
-            xpath = "//x:FareZone"
-            zones = fare_zones[0].xpath(xpath, namespaces=NAMESPACE)
-            members_list = []
-            for zone in zones:
-                xpath = "x:members"
-                members = zone.xpath(xpath, namespaces=NAMESPACE)
-                if members:
-                    members_list.append(members)
-                    continue
-                elif not members:
-                    sourceline_zone = zone.sourceline
-                    response_details = XMLViolationDetail(
-                        "violation",
-                        sourceline_zone,
-                        msg.MESSAGE_OBSERVATION_FARE_ZONES_MEMBERS_MISSING,
-                    )
-                    response = response_details.__list__()
-                    return response
-            for members in members_list:
-                xpath = "x:ScheduledStopPointRef"
-                schedule_stop_points = members[0].xpath(xpath, namespaces=NAMESPACE)
-                sourceline_schedule_stop_points = members[0].sourceline
-                if not schedule_stop_points:
-                    response_details = XMLViolationDetail(
-                        "violation",
-                        sourceline_schedule_stop_points,
-                        msg.MESSAGE_OBSERVATION_SCHEDULED_STOP_POINT_REF_MISSING,
```

Comment:
Please remove the definition for this and other validation message constants.

Reply:
change done, pls check